databuilder/backends/base.py

```diff
@@ -55,7 +55,7 @@ def get_table_expression(self, table_name):
         Args:
             table_name: Name of Table
         Returns:
-            A SQL subquery
+            A SQLAlchmey TableClause
```

Comment:
Ah, I don't actually do this because I find it useful. It's more that someone else added the comments and stripping them out completely feels a bit brutal; but I can't leave them as lies, so I make the minimal change to keep them accurate.

I think this would be a perfect use for type signatures. The fact that strictly typing everything didn't work out for us shouldn't mean that we can't use them at all, I think.

I'd be happy to start adding types in as we go, wherever doing so aids readability and comprehension.